### ISSUE TITLE
Limit numbers of Items to be shown

### ISSUE THREAD

**Issue by X-Tender** [haltu/muuri#462]
Is it possible to limit the numbers of items shwon? Like I have 50 items but only want that 3 of them are shown in combination with a filter (ever item have a category which can be filtered for).
But I only want that the first 3 items are displayed.

**Comment by thednp** [comment]
As far as I've read the code so far, I think this would be possible with some special filter function.

A quick mockup would possibly look like this:

```js
    grid.filter(
      (item) => {
        const element = item.getElement();
        const isSearchMatch = !searchFieldValue
          || (element.getAttribute('data-title') || '').toLowerCase().indexOf(searchFieldValue) > -1);
        const isFilterMatch = !filterFieldValue
          || (filterFieldValue === element.getAttribute(`data-title`));
        const isBelowIndex = OTHER_CONDITION && grid.items.indexOf(element) > YOUR_LIMIT;
        return isSearchMatch && isFilterMatch && isBelowIndex;
      },
      { onFinish },
    );
```

Man I love this library.

**Comment by X-Tender** [comment]
Without testing it, but wouldn't it check the index of all items? not the already filtered? Like when I have 10 Items, First 5 have a value age = 10 and the other 5 have age = 10. And when I filter it out for items with age > 5 then the isBelowIndex would always return false when I say YOUR_LIMIT ist 3. Because the items with age > 5 are starting at index 4.

**Comment by thednp** [comment]
Indeed, but you get the idea. the `OTHER_CONDITION` is there to make sure you get what you want.

**Comment by X-Tender** [comment]
Is there maybe a property that contains the filtered/visible items? Or it it would be possible to chain the filter method :D

**Comment by thednp** [comment]
Yes there is, if you `console.log(grid._items)`, you can inspect the object properties and make up your `OTHER_CONDITION`.
